test/stickerbook.test.js

```diff
@@ -290,6 +290,32 @@ describe('Stickerbook', () => {
     stickerbook.off('mouse:down', callback);
   });
 
+  it('cannot place a sticker without a position', (done) => {
+    const stickerbook = createStickerbook();
+
+    stickerbook.setSticker('http://www.example.com/images/A.png')
+      .then(() => {
+        try {
+          stickerbook.placeImage({});
+        } catch(e) {
+          return done(); // we want it to err!
```

Comment:
can we check the error message in this test to make sure it's erroring "in the right way"?